substrate/frame/bags-list/src/lib.rs

```diff
@@ -234,6 +235,14 @@ pub mod pallet {
 		#[pallet::constant]
 		type BagThresholds: Get<&'static [Self::Score]>;
 
+		/// Maximum number of accounts that may be re-bagged automatically in `on_idle`.
+		///
+		/// A value of `0` (obtained by configuring `type AutoRebagPerBlock = ();`) disables
+		/// the feature completely and keeps the pallet’s behaviour identical to pre-upgrade
```

Comment:
```suggestion
		/// the feature. 
```

I think this is clearer because when you say `pre-upgrade`, it is unclear to me which upgrade you are referring to.